Bluetooth: Mesh: Remove storage timers from models (#11517)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

Bluetooth mesh
--------------

> * Updated:

Reviewer: Move the entry under Libraries:

E.g.
Bluetooth libraries and services
--------------------------------

* :ref:`bt_mesh` library:

  * Updated:

    * The :kconfig:option:`BT_MESH_MODEL_SRV_STORE_TIMEOUT` Kconfig option, that is controlling timeout for storing of model states, is replaced by the :kconfig:option:`BT_MESH_STORE_TIMEOUT` Kconfig option.

>   * The :kconfig:option:`BT_MESH_MODEL_SRV_STORE_TIMEOUT` Kconfig option have been removed.
>     To control timeout for storing models states, use the :kconfig:option:`BT_MESH_STORE_TIMEOUT` Kconfig option instead.

Reviewer: ```suggestion
  * The :kconfig:option:`BT_MESH_MODEL_SRV_STORE_TIMEOUT` Kconfig option, that is controlling timeout for storing of model states, is replaced by the :kconfig:option:`BT_MESH_STORE_TIMEOUT` Kconfig option.
```
